public/docs/dart/latest/guide/forms.jade

```diff
@@ -0,0 +1,662 @@
+    we should assign to the model's `name` property.
+
+    Nice to know but is it practical? `[(ng-model)]` is usually what we want, but
+    we might split the binding when the event handling has to do something special
+    such as debounce or throttle the keystrokes.
+
+    <!-- TODO: Add the following once template-syntax.html exists.
+    Learn more about `NgModel` and other template syntax in the
+    [Template Syntax](./template-syntax.html) guide.
+    -->
+
+
+.l-main-section
+:marked
+  ## Track change-state and validity with **ng-control**
+
+  A form isn't just about data binding. We'd also like to know the state of the controls on our form.
+  The `NgControl` directive keeps track of control state for us.
+
+.callout.is-helpful
+  header NgControl requires Form
+  :marked
+    The `NgControl` is one of a family of `NgForm` directives that can only be applied to
+    a control within a `<form`> tag.
+:marked
+  Our application can ask an `NgControl` instance whether
+  the user touched the control, the value changed, or the value is valid.
+
+  `NgControl` doesn't just track state; it updates the control with special
+  Angular CSS classes, such as `ng-valid` or `ng-invalid`.
+  We can use those class names to change the appearance of the
+  control and make messages appear or disappear.
+
+  We'll explore those effects soon. Right now
+  let's **add `ng-control`to all three form controls**,
+  starting with the Name input box.
++makeExample('forms/dart/angular2_forms-6-NGCONTROL/lib/hero_form_component.html', 'ng-control-1', 'lib/hero_form_component.html')(format="linenums:6")
+:marked
+  Be sure to assign a unique name to each `ng-control` directive.
+
+.l-sub-section
+  :marked
+    Angular registers controls under their `ng-control` names
+    with the `NgForm` directive.
+    We didn't add the `NgForm` directive explicitly but it's here;
+    we'll talk it [later in this guide](#ng-form).
+
+
+.l-main-section
+  h2 Add custom CSS for visual feedback
+
+  p.
+    <code>NgControl</code> doesn't just track state. It updates the control to
+    have three classes that describe the control's state.
+
+  table
+    tr
+      th State
+      th Class if true
+      th Class if false
+    tr
+      td Control has been visited
+      td <code>ng-touched</code>
+      td <code>ng-untouched</code>
+    tr
+      td Control's value has changed
+      td <code>ng-dirty</code>
+      td <code>ng-pristine</code>
+    tr
+      td Control's value is valid
+      td <code>ng-valid</code>
+      td <code>ng-invalid</code>
+
+:marked
+  Let's add a temporary local template variable
+  <!-- TODO: make that a link to (./template-syntax.html#local-vars) -->
+  named **spy**
+  to the Name `<input>` tag and use the spy to display those classes.
+
++makeExample('forms/dart/angular2_forms-7.1-CSS-SPY/lib/hero_form_component.html', 'ng-control-2')(format="linenums:6")
+
+:marked
+  Now let's run the app, focusing our attention on the Name input box
```

Comment:
I do like it better, although I worry that "focus" could be misconstrued as setting the place where keyboard input goes. How about:

  Now run the app, and look at the Name input box.
  Follow the next four steps *precisely*:

Also... do we need to italicize UI names? I was relying on capitalization alone to set them apart. (I once had feedback that many people have trouble reading italics, so I avoid them whenever possible. People seem to read boldface more easily, but I try to use it sparingly since it attracts so much attention.)